static/js/topic_page/map_tile.tsx

```diff
@@ -60,63 +56,65 @@ interface RawData {
   placeStat: PlacePointStat;
   metadataMap: Record<string, StatMetadata>;
   population: StatApiResponse;
-  place: NamedTypedPlace;
   parentPlaces: NamedTypedPlace[];
 }
 
-interface MapChartData {
+interface ChartData {
```

Comment:
i renamed this from ChartData to MapChartData since we had one per component and it gets confusing as you work across tiles (also makes search a little harder). it's inconsistent now across tiles, though, so we can keep it one way or the other.

Reply:
done